src/features/stock-out.js

```diff
@@ -279,12 +287,34 @@ async function getStockOutConfigs({
           value: 'weekly_qty'
         }
       ],
+      when: function (answers){
+        const argv = process.argv;
+        if (!argv[6]){
+          return true;
+        } 
+        answers.formular = argv[6];
+        return false;
+      }
     },
     ...languages.map((language) => ({
       type: 'input',
       name: `title.${language}`,
       message: `Enter stock out form title in ${language}`,
-      default: 'Stock Out'
+      default: 'Stock Out',
+      when: function (answers){
+        const argv = process.argv;
+        if (!argv[7]){
+          return true;
+        }  
+        const answer = {
+          title:{
+            'en': argv[7].split(',')[0],
```

Comment:
From my previous comment I think it will be better to:

`
title: {
    'en': validator.escape(argv[7].split(',')[0]),
     'fr': validator.escape(argv[7].split(',')[1])
}
`

If you accept this, please update in the other files